### ISSUE TITLE
[feature request] handle modded woods like vanilla wood with saw

### ISSUE THREAD

**Issue by mvdhoning** [An-Sar/PrimalCore#121]
In my modpack i set up primal core to be needing the saw to get planks from logs.
That works like a charm for vanilla wood types.
But not for logs added by other mods like forestry, binnies, twilight forest and more.

Maybe if i really wanted to put effort into it i could manualy add craftweaker entries of each kind of wood and each kind of saw but that would be a PITA if possible at all as removing the basic recipe makes one need to manually re-add all other mods special woordworking machines like binnies sawtable.

Hope you can find an easy way to handle other mods woods log to planks like you did for vanilla woods.

**Comment by An-Sar** [comment]
i agree this would be a good support feature to have, i am not entirely sure if there is a good way to do without causing issues related to just removing recipes. 

worst case it could come down to some official tweaker configs maybe, but it would be nice to have something built in.

**Comment by Efimero** [comment]
OP: remove recipes by name, that way you can cherry pick which versions to remove instead of blanketing them, and use oreDict and loops to make your life easier =3 <3

**Comment by belathus** [comment]
I'll just share a script I made via crafttweaker for this: https://github.com/belathus/Wanderlust-Renewed/blob/master/scripts/planks.zs

**Comment by mvdhoning** [comment]
thanks @belathus for your script!
that does just what i wanted
with that as a base you can also add binnies woods
even after removing the shapeless recipe this way other mods stay to keep their recipes

**Comment by RiverC** [comment]
Y'all mind if I... update the script (being used for my modpack) to fix the name change in Rockhounding, add primal yew and ironwood, and add ALL of current Binnie's Mods Extra Trees woods?

https://pastebin.com/9Ea2v6vP

For some reason, Primal Yew does not seem to have stairs.
